contract_manager/scripts/common.ts

```diff
@@ -212,3 +219,110 @@ export function findEvmChain(chainName: string): EvmChain {
   }
   return chain;
 }
+
+/**
+ * Finds the wormhole contract for a given EVM chain.
+ * @param {EvmChain} chain The EVM chain to find the wormhole contract for.
+ * @returns If found, the wormhole contract for the given EVM chain. Else, undefined
+ */
+export function findWormholeContract(
+  chain: EvmChain
+): EvmWormholeContract | undefined {
+  for (const contract of Object.values(DefaultStore.wormhole_contracts)) {
+    if (
+      contract instanceof EvmWormholeContract &&
+      contract.getChain().getId() === chain.getId()
+    ) {
+      return contract;
+    }
+  }
+}
+
+interface DeployWormholeReceiverContractsConfig {
```

Comment:
we can make this interface extend the base DeployConfig and just add `type` to it.